packages/compass-aggregations/src/components/stage-toolbar/index.tsx

```diff
@@ -129,10 +136,11 @@ export function StageToolbar({
       <div className={leftStyles}>
         <StageCollapser index={index} />
         <Body weight="medium">Stage {idxInPipeline + 1}</Body>
-        <div className={selectStyles}>
+        <div className={spacingRightStyles}>
```

Comment:
Not sure how much this matches the expected design, but whatever is happening with the spacing between elements here looks a bit off to me: half of the elements have one gap size, half add a margin on the right to it? Maybe we can just set the same, bigger gap for everything here and remove this additional class name?

Seems like maybe the idea was to have `stage n` label closer to the stage operator select and everything else spaced evenly, maybe worth wrapping them in a container so that their gap can be separate?

Reply:
I addressed this as well in [4e8288d](https://github.com/mongodb-js/compass/pull/4564/commits/4e8288df0ccfa3cf037d8ba13127b1e457dc53d4)